h/services/annotation_moderation.py

```diff
@@ -167,6 +167,8 @@ def queue_moderation_change_email(self, request, moderation_log_id: int) -> None
             html=render(f"{template_base}.html.jinja2", context, request=request),
             tag=EmailTag.MODERATION,
             subaccount=self._email_subaccount,
+            reply_to=group.reply_to,
+            from_name=group.email_from_name,
```

Comment:
This passes through the group's `reply_to` and `email_from_name` (if any) when sending moderation notification emails. The new fields are not used for any other types of email yet.